DevCycle.SDK.Server.Common/Model/Local/Variable.cs

```diff
@@ -104,29 +102,27 @@ public static Variable<T> InitializeFromVariable(Variable<T> variable, string ke
         /// <value>Unique key by Project, can be used in the SDK / API to reference by &#x27;key&#x27; rather than _id.</value>
         [DataMember(Name = "key")]
         public string Key { get; set; }
-        
-        [DataMember(Name = "isDefaulted")]
-        public bool IsDefaulted { get; set; }
+
+        [DataMember(Name = "isDefaulted")] public bool IsDefaulted { get; set; }
```

Comment:
same here, please keep the formatting consistent.

Reply:
This was an auto-lint fix - not something I did myself

Reply:
why is the formatting different between the params then?

Reply:
Not sure - that's something to ask resharper.

Reply:
we can skip fixing this if its actually linting, but doesn't really make sense to me why this would be treated any differently than the other lines above.